src/index.html

```diff
@@ -17,11 +18,372 @@
+
+        <section class="card__product-code">Product code: 195434</section>
+
+        <section class="card__stars stars">
+          <div class="stars__stars-box">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="stars__reviews">Reviews: 5</div>
+        </section>
+
+        <section class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-price">$2,199</div>
+        </section>
+
+        <section class="card__button">
+          <div
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </div>
+        </section>
+      </div>
+      <div
+        class="card main__card"
+        data-qa="card"
+      >
+        <section class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </section>
+
+        <section class="card__product-code">Product code: 195434</section>
+
+        <section class="card__stars stars">
+          <div class="stars__stars-box">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="stars__reviews">Reviews: 5</div>
+        </section>
+
+        <section class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-price">$2,199</div>
+        </section>
+
+        <section class="card__button">
+          <div
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </div>
+        </section>
+      </div>
+      <div
+        class="card main__card"
+        data-qa="card"
+      >
+        <section class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </section>
+
+        <section class="card__product-code">Product code: 195434</section>
+
+        <section class="card__stars stars">
+          <div class="stars__stars-box">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="stars__reviews">Reviews: 5</div>
+        </section>
+
+        <section class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-price">$2,199</div>
+        </section>
+
+        <section class="card__button">
+          <div
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </div>
+        </section>
+      </div>
```

Comment:
The repeated sections for the product cards could be optimized by using a templating engine or JavaScript to dynamically generate these sections. This would reduce redundancy and make the code easier to maintain.